docs/build/index.mdx

```diff
@@ -6,7 +6,4 @@ description: Build circuits with Qiskit and Qiskit Runtime
 
 # Build circuits with Qiskit
 
-In the build phase, you create quantum programs that represent the problem you are solving. The foundation of quantum programs are quantum circuits, which are made up of gates (or operations) that are used to manipulate qubits in the quantum computer. You can also use _primitives_, which provide methods that make it easier to build modular algorithms and other higher-order programs, as well as providing the interface to run those circuits on a backend and get outputs in a specific format.  Depending on your needs, you can use Qiskit or OpenQASM to create your circuits, followed by Qiskit Runtime to initialize your chosen primitive for an IBM Quantum backend. In summary, the build phase consists of these processes:
-
-1. Defining quantum circuits
-2. Initializing primitives
+In the build phase, you create quantum programs that represent the problem you are solving. The foundation of quantum programs are quantum circuits, which are made up of gates (or operations) that are used to manipulate qubits in the quantum computer.  Depending on your needs, you can use Qiskit or OpenQASM to create your circuits.
```

Comment:
This intro is good for me but @blakejohnson do you have thoughts?

Reply:
It does feel a little short now that we've removed the primitives stuff 🤔

Reply:
Along the same lines, we may want to add primitives info to the Run introduction page -- or should it stay the same? Here's how it looks now
<img width="874" alt="Screenshot 2023-10-10 at 12 16 31 PM" src="https://github.com/Qiskit/documentation/assets/28769477/420e1f32-1b0e-46cb-9188-bd29032d15d3">

Reply:
Gates are not the only valid operations in a quantum circuits. So, I would suggest something to the effect of:
> are made up of operations including gates, measurement, and reset of qubits.

Reply:
Ok, so some other things I think we need to do in the intro are to talk about what it is we are building. This effectively should signpost what is to come in the section. i.e. something like:

All tasks require building one or more quantum circuits. Some tasks additionally require constructing quantum operators to define properties of quantum states that we want to estimate/measure.

Qiskit enables working with circuits (and, to a lesser extent, operators) at various abstraction levels:
* abstract
* virtual
* physical
* scheduled
* pulse program
At the most **abstract** level, we have a task-oriented lens in the circuit library. We can also express operations in abstract mathematical terms using Operators, isometries, and classical/boolean functions. Then we have **virtual** circuits where those mathematical abstractions take on a concrete representation in terms of a concrete gate set. At the **physical** level those instructions are mapped to specific physical qubits and instructions are re-written to reflect the connectivity and native gate set of a target hardware platform. **Scheduled** circuits introduce timing information. And finally **pulse programs** represent signals on channels. 

Qiskit, and OpenQASM, further support the notion of _extended_ circuits which expand the set of allowed operations to include real-time computations on classical values. Qiskit's tooling for working with this richer family of circuits is found in the section on Dynamic Circuits.

Reply:
Thanks Blake -- moving your comments into #143 now that we have an intro-page-specific PR.